### PR TITLE
Feature/dynamic tables

### PR DESCRIPTION
**Pull request by ArminWindlin**

### CODE REVIEW

File: src/views/article.vue [hunk 1]
```diff
@@ -109,6 +120,20 @@ export default {
             }
             return style;
         },
+        sortTable(table, column) {
+            let revMlt = 1;
```
Comment (Zahikusa) [inline src/views/article.vue:124]: What does ``revMlt`` stand for here?

File: src/data/articles/units/units.json [hunk 2]
```diff
@@ -4,15 +4,157 @@
...
+          "Recruitment Time",
+          "Speed (s / field)"
+        ],
+        "rows": [
+          [
+            "{{ units.raiderHero.name }}",
+            "{{ units.raiderHero.attack }}",
+            "{{ units.raiderHero.defense }}",
+            "{{ units.raiderHero.recSpeed }}",
+            "{{ units.raiderHero.speed }}"
```
Comment (Zahikusa) [inline src/data/articles/units/units.json:140]: So far i can't find this information visually displayed in the "units-overview" of the wiki. Is this intentionally not yet displayed and just does to show how dynamic tables are used?

**Comment by Zahikusa** [comment]
image
Looks a bit off with so much space on the right side of the table imo. Maybe this only happens on my wide-screen.

**Comment by Zahikusa** [comment]
Overall another cool addition that improves productivity for maintenance and at the same time user experience on our Orb. Nice!

File: src/views/article.vue (hunk 1 above)
Comment (ArminWindlin) [reply src/views/article.vue]: reverse multiplier (to reverse the order with a multiplier). I adjusted the name now 👍

Comment (ArminWindlin) [reply src/views/article.vue]: > image
> Looks a bit off with so much space on the right side of the table imo. Maybe this only happens on my wide-screen.

Adjusted it that table take full space within container

Comment (Zahikusa) [reply src/views/article.vue]: Thanks for explaining.

**Review by Zahikusa — APPROVED** [review]

**Pull request merged**